frontend/public/components/graphs/donut.jsx

```diff
@@ -23,8 +23,8 @@ export class Donut extends SafetyFirst {
     this.resize = () => this.node && Plots.resize(this.node);
 
     this.layout = {
-      height: 150,
-      minWidth: 150,
+      height: 300,
+      minWidth: 300,
```

Comment:
We can't make this change globally. This affects all usage of the donut.

Reply:
Ack. Will find another way to implement the size change, probably by passing another variable.